testing/python/cuda/test_tilelang_cuda_tcgen05_operand_layout.py

```
@@ -0,0 +1,369 @@
+def _swizzle_128b(shape, plain_address):
+    """Construct BF16 ``Sw<3,3,3> o plain_address`` for layout probes."""
+
+    def forward(*coords):
+        address = plain_address(*coords)
+        return address ^ ((address & (7 << 6)) >> 3)
+
+    return Layout(shape, forward)
+
+
+def _ranges(mins, extents):
+    return [tvm.ir.Range.from_min_extent(begin, extent) for begin, extent in zip(mins, extents)]
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)